RELEASES.md

```diff
@@ -0,0 +1,149 @@
+# Releases
+
+Tendermint uses [semantic versioning](https://semver.org/) with each release following
+a `vX.Y.Z` format. The `master` branch is used for active development and thus it's
+advisable not to build against it.
+
+Releases are specified using tags and are built from long-lived "backport" branches.
+Each release "line" (e.g. 0.34 or 0.33) has its own long-lived backport branch,
+and the backport branches have names like `v0.34.x` or `v0.33.x`
+(literally, `x`; it is not a placeholder in this case). Tendermint only maintains the last two releases at a time (the oldest release is predominantly just security patches).
+
+## Backporting
+
+As non-breaking changes land on `master`, they should also be backported (cherry-picked)
+to these backport branches.
+
+We use Mergify's [backport feature](https://mergify.io/features/backports) to automatically backport
+to the needed branch. There should be a label for any backport branch that you'll be targeting.
+To notify the bot to backport a pull request, mark the pull request with
+the label `S:backport-to-<backport_branch>`.
```

Comment:
```suggestion
For example, to backport to v0.34.x, add the label `S:backport-to-v0.34.x`.
```

Advice from the technical writing team at my $LASTJOB: people are often pattern matching in their heads as they read through docs so it's useful to just give an example instead of doing a sort of pseudo-templating in documentation.